boa/network.py

```diff
@@ -43,8 +43,7 @@ def returndata_bytes(self):
     def is_error(self):
         if "structLogs" in self.raw_trace:
             return self.raw_trace["failed"]
-        else:
-            return "error" in self.raw_trace
+        return self.raw_trace.get("error") is not None
```

Comment:
looks like a semantics-changing change. why is this necessary?

Reply:
zksync always returns `"error=None"`

Reply:
spooky. in that case, comment would be good
```suggestion
        # we can have `"error": null` in the payload
        return self.raw_trace.get("error") is not None
```